### ISSUE TITLE
no device found, ensure the device is not locked

### ISSUE THREAD

**Issue by Alexbossnice** [kiwi-cam/homebridge-broadlink-rm#575]
**Describe the bug**
no device found, ensure the device is not locked : error appear with last firmware of broad link : v62093

devine can't connect.

No change of home bridge setup.

**To Reproduce**
firmware update of rm pro v62093

**Comment by petykowski** [comment]
I am also experiencing the same issue as of v62093 firmware on my RM 4 PRO.

**Comment by alexbrazier** [comment]
Is there any way to rollback the firmware? Seems there is an manual update button but you need the download url.

**Edit**
Found the link to the firmware [here](https://www.reddit.com/r/homebridge/comments/1181ex4/caution_to_people_using_broadlink_rm_plugin/). Downgraded my RM4 pro from `v62093` to `v62092` and after restarting Hombridge it started working again. No need to reset/remove from the Broadlink app, just make sure it isn't locked.

**Comment by alexbrazier** [comment]
I've debugged the issue as far as here where the socket never response with a `message` on device connection.

https://github.com/kiwi-cam/broadlinkjs-rm/blob/23272b047d50759df7de3ddd30181df7b472f87c/index.js#L311

I also added a `socket.on('error', ...)` listener which doesn't log anything.

Not sure if this is an issue with `v62093` that they will rollback, or an issue that the library should be handling.
